samples/samples/snippets.py

```diff
@@ -306,16 +348,57 @@ def update_database_with_default_leader(instance_id, database_id, default_leader
 # [END spanner_update_database_with_default_leader]
 
 
+# [START spanner_update_database_with_proto_descriptors]
+def update_database_with_proto_descriptors(instance_id, database_id):
+    """Updates a database with tables with a default leader."""
+    spanner_client = spanner.Client(client_options={"api_endpoint": "staging-wrenchworks.sandbox.googleapis.com"})
+    instance = spanner_client.instance(instance_id)
+
+    database = instance.database(database_id)
+    proto_descriptor_file = open("testdata/descriptors.pb", 'rb').read()
+
+    operation = database.update_ddl(
+        [
+            """CREATE PROTO BUNDLE (
+            spanner.examples.music.SingerInfo,
+            spanner.examples.music.Genre,
+            )""",
+            """CREATE TABLE SingersProto (
+            SingerId   INT64 NOT NULL,
+            FirstName  STRING(1024),
+            LastName   STRING(1024),
+            SingerInfo spanner.examples.music.SingerInfo,
+            SingerGenre spanner.examples.music.Genre,
+            ) PRIMARY KEY (SingerId)""",
+        ],
+        proto_descriptors=proto_descriptor_file
+    )
+    print("Waiting for operation to complete...")
+    operation.result(OPERATION_TIMEOUT_SECONDS)
+
+    database.reload()
+
+    print(
+        "Database {} updated with proto descriptors".format(
+            database.name
+        )
+    )
+
+
+# [END spanner_update_database_with_proto_descriptors]
+
+
 # [START spanner_get_database_ddl]
 def get_database_ddl(instance_id, database_id):
     """Gets the database DDL statements."""
-    spanner_client = spanner.Client()
+    spanner_client = spanner.Client(client_options={"api_endpoint": "staging-wrenchworks.sandbox.googleapis.com"})
     instance = spanner_client.instance(instance_id)
     database = instance.database(database_id)
     ddl = spanner_client.database_admin_api.get_database_ddl(database=database.name)
     print("Retrieved database DDL for {}".format(database_id))
     for statement in ddl.statements:
         print(statement)
+    print(ddl.proto_descriptors)
```

Comment:
Change checks in snippets_test.py to check for correct value

Reply:
`ddl.proto_descriptors` is of type bytes and it would be difficult to check for correct value. Do you prefer removing this line instead?

Reply:
You don't have to check the whole value, just check for a keyword that would be present in the output

Reply:
the output is a byte string which is not very readable. let me figure it out.